baremetal/metal3remediation_manager.go

```diff
@@ -38,15 +39,20 @@ import (
 )
 
 const (
-	rebootAnnotation = "reboot.metal3.io"
```

Comment:
I'm not sure you can change this... its specified by the metal3 API

Reply:
I understand when I change it in the way I did it will do a power off instead o a reboot?
```
The basic annotation form (reboot.metal3.io) triggers the controller to power cycle the Host. This form has set-and-forget semantics and the controller removes the annotation once it restores power to the Host.

An advanced form (reboot.metal3.io/{key}) instructs the controller hold the Host in a PoweredOff state so that the caller can perform any required actions while the node is in a known safe state.
```

https://github.com/metal3-io/metal3-docs/blob/c5608f6540e2f542da22b43e051c6c176ea08ef9/design/baremetal-operator/reboot-interface.md#proposal